lib/main.js

```diff
@@ -21,40 +22,55 @@ export default class LinterUI {
   statusBar: StatusBar;
   intentions: Intentions;
   subscriptions: CompositeDisposable;
+  idleCallbacks: Set<*>;
```

Comment:
Are you sure these are numbers? The documentation just says "handle", as I'm not sure exactly what type that really is, I left it as any.